tests/Tools/generate-stub-map

```diff
@@ -243,11 +244,11 @@ use const PHP_EOL;
         }
     }
 
-    $mapWithRelativeFilePaths = array_map(static function (array $files) use ($phpStormStubsDirectory) : array {
+    $mapWithRelativeFilePaths = array_map(static function (array $files) use ($mapFile) : array {
         ksort($files);
 
-        return array_map(static function (string $filePath) use ($phpStormStubsDirectory) : string {
-            return str_replace('\\', '/', substr($filePath, strlen($phpStormStubsDirectory)));
+        return array_map(static function (string $filePath) use ($mapFile) : string {
+            return str_replace('\\', '/', substr($filePath, strlen(dirname($mapFile)) + 1));
```

Comment:
Presumably, the path that we want should be relative to the map file, not the stub directory.
This works (better) in the current case, but it would be a bad solution in case of diverging paths.